force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,26 +177,56 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
-    }
+        String gradeResult;
+
+        if (score >= 90) {
+            gradeResult = 'A';
+        }
+        if (score >= 80 && score <= 89) {
+            gradeResult = 'B';
+        }
+        if (score >= 70 && score <= 79) {
+            gradeResult = 'C';
+        }
+        if (score >= 60 && score <= 69) {
+            gradeResult = 'D';
+        }
+        if (score <= 59) {
+            gradeResult = 'F';
+        }
```

Comment:
For practice try reversing the order of the if statement values. 
gradeResult = F is the if clause